src/storage/src/healthcheck.rs

```diff
@@ -118,24 +125,181 @@ pub(crate) struct ObjectHealthConfig {
     pub(crate) persist_location: PersistLocation,
 }
 
-struct HealthState<'a, O> {
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+pub enum StatusNamespace {
```

Comment:
💭 I wonder if it makes sense to encode the worker id in the namespace... that way namespaces that don't have worker ids (like the ssh tunnel stuff) get to ignore them.

Reply:
I would do this, but it would be too onerous on the caller i think...the new code im about to pushup makes this a tiny bit cleaner i think